ci/docker/python-wheel-windows-test-vs2019.dockerfile

```
@@ -20,44 +20,21 @@
 
 # based on mcr.microsoft.com/windows/servercore:ltsc2019
 # contains choco and vs2019 preinstalled
-FROM abrarov/msvc-2019:2.11.0
+ARG base
+# hadolint ignore=DL3006
+FROM ${base}
 
 # hadolint shell=cmd.exe
 
-# Add unix tools to path
-RUN setx path "%path%;C:\Program Files\Git\usr\bin"
-
-# 1. Remove previous installations of python from the base image
-# NOTE: a more recent base image (tried with 2.12.1) comes with python 3.9.7
-# and the msi installers are failing to remove pip and tcl/tk "products" making
-# the subsequent choco python installation step failing for installing python
-# version 3.9.* due to existing python version
-# 2. Install Minio for S3 testing.
-RUN wmic product where "name like 'python%%'" call uninstall /nointeractive && \
-    rm -rf Python* && \
-    curl https://dl.min.io/server/minio/release/windows-amd64/archive/minio.RELEASE.2024-09-13T20-26-02Z \
-        --output "C:\Windows\Minio.exe"
-
-# Install the GCS testbench using a well-known Python version.
-# NOTE: cannot use pipx's `--fetch-missing-python` because of
-# https://github.com/pypa/pipx/issues/1521, therefore download Python ourselves.
-RUN choco install -r -y --pre --no-progress python --version=3.11.9
-ENV PIPX_BIN_DIR=C:\\Windows\\
-ENV PIPX_PYTHON="C:\Python311\python.exe"
-COPY ci/scripts/install_gcs_testbench.bat C:/arrow/ci/scripts/
-RUN call "C:\arrow\ci\scripts\install_gcs_testbench.bat" && \
-    storage-testbench -h
-
 # Define the full version number otherwise choco falls back to patch number 0 (3.9 => 3.9.0)
 ARG python=3.9
 RUN (if "%python%"=="3.9" setx PYTHON_VERSION "3.9.13") & \
     (if "%python%"=="3.10" setx PYTHON_VERSION "3.10.11") & \
     (if "%python%"=="3.11" setx PYTHON_VERSION "3.11.9") & \
     (if "%python%"=="3.12" setx PYTHON_VERSION "3.12.5") & \
-    (if "%python%"=="3.13" setx PYTHON_VERSION "3.13.0-rc1")
+    (if "%python%"=="3.13" setx PYTHON_VERSION "3.13.0-rc2")
```

It seems this is available on [chocolatey now](https://community.chocolatey.org/packages/python313)
```suggestion
    (if "%python%"=="3.13" setx PYTHON_VERSION "3.13.1")
```
We can also update on:
[ci/docker/python-wheel-windows-vs2019.dockerfile](https://github.com/apache/arrow/pull/44804/files#diff-33b7860d19ff7ce9bdfe7f8f623dedab496b9d2fa3727ac57a7a9d6831cb23ea)